app/js/components/SortableTable/index.js

```diff
@@ -0,0 +1,219 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
```

Comment:
@mogoodrich, so @jomadoye and I had a discussion around routing and if we'd be making use of [react-router](https://reacttraining.com/react-router/) in this project and we came to conclusion that this was needed. by just looking up the mockups we noticed that the views had some sort of streamlined paths based on the breadcrumbs and if we didn't incorporate routing we would run into issues trying to organise the views and their child paths especially when we have had a ton of work done. so from that thinking I was thinking of creating a `Views` or `Pages` folder in `app/js/` and then those would be folders having different views or pages pertaining to the different locations in the app and then the component folder would just hold reusable components across the entire app,  that's why I put it in it's own folder and used that convention, but I agree in renaming it to just `SortableTable.jsx` for now since we're concentrating on just the lab results page

Reply:
it seems @jomadoye created a shared folder where he put the breadcrumbs component, so that works fine also

Reply:
@mogoodrich It's currently in `components/shared/SortableTable.jsx` where the breadcrumb is just incase we need to reuse it, what do you think ?

Reply:
So maybe "shared" is where we put components we think would be good candidates to migrate to openmrs-react-components, and "components" is for components "shared" across the lab workflow app, and a new directory like "pages" or "views" would be for actual routes?

Reply:
@mogoodrich yeah that makes sense, @jomadoye what do you think ?